### PR TITLE
Implementation of the dual decomposition of stochastic MIQP

### PR DESCRIPTION
**Pull request by cheesecakeball**

### CODE REVIEW

File: src/DspCInterface.h [hunk 1]
```diff
@@ -62,6 +62,19 @@ int readDro(DspApiEnv * env, const char * dro);
...
+ 		const double *       value, /**< constraint elements */
+ 		const double *       clbd,  /**< column lower bounds */
+ 		const double *       cubd,  /**< column upper bounds */
+ 		const char *         ctype, /**< column types */
+ 		const double *       obj,   /**< objective coefficients */
+ 		const double *       rlbd,  /**< row lower bounds */
+ 		const double *       rubd   /**< row upper bounds */);
+
+/** load first-stage problem with quadratic objective */
+void loadQuadraticFirstStage(
```
Comment (kibaekkim) [inline src/DspCInterface.h:77]: Can we overload function `loadFirstStage`?

File: src/DspCInterface.h [hunk 2]
```diff
@@ -70,11 +83,30 @@ void loadFirstStage(
...
+ 		const double *       value, /**< constraint elements */
+ 		const double *       clbd,  /**< column lower bounds */
+ 		const double *       cubd,  /**< column upper bounds */
+ 		const char *         ctype, /**< column types */
+ 		const double *       obj,   /**< objective coefficients */
+ 		const double *       rlbd,  /**< row lower bounds */
+ 		const double *       rubd   /**< row upper bounds */);
+
+/** load second-stage problem */
+void loadQuadraticSecondStage(
```
Comment (kibaekkim) [inline src/DspCInterface.h:109]: Can we overload function `loadSecondStage`?

File: src/DspCInterface.h (hunk 1 above)
Comment (cheesecakeball) [reply src/DspCInterface.h:77]: Since this is C interface, overload is not allowed. We may use C++ for DspCinterface, then we can overload functions.

**Comment by cheesecakeball** [comment]
This is the complete code for solving the extensive formulation of TSS-MIQP. I am working on the Benders Decomposition and the callback function layer on my branch. If this pull request is passed, I can create another pull request for dual decomposition of TSS-MIQP without cuts (which is also done).

**Comment by kibaekkim** [comment]
This PR failed in the Travis CI test. Can you look into and fix that?

**Comment by cheesecakeball** [comment]
> This PR failed in the Travis CI test. Can you look into and fix that?

The PR passed the request now.

**Pull request merged**